### ISSUE TITLE
Endpoint ApiKeyRequired

### ISSUE THREAD

**Issue by DanDiac** [jaws-framework/JAWS#208]
Setting ApiKeyRequired to true in awsm.json doesn't work: the aws API GATEWAY endpoints are updated, and the value of API Key Required is set to false every time (even if it was manually set to true before running jaws dash).

**Comment by lukewpatterson** [comment]
Do you know when this will be in the released code?  I just finished a proof-of-concept migration of an existing custom script-based project, and this is the very last item on my list of things to check.  JAWS is great, really looking forward to being able to use it and evangelize it.

**Comment by ac360** [comment]
@lukewpatterson I just fixed that in v1.4 this morning.  We're trying to get v1.4 out by the end of this weekend.  Please stay tuned, even bigger project updates are coming :)
